### ISSUE TITLE
How to make predictions on test set ?

### ISSUE THREAD

**Issue by shraddha291996** [sicara/easy-few-shot-learning#75]
Hello,

Is there a way to make predtictions on test set in evaluation phase , in your notebook in episodic training and classical training you calculate the accuracy but I also want to predict the class on test set. Can you please suggest regarding this ?
Thanks in advance

**Comment by ebennequin** [comment]
I believe this is the same question as in #17 .

As you can see in the [source code](https://github.com/sicara/easy-few-shot-learning/blob/master/easyfsl/methods/utils.py) for the `evaluate()` method, prediction is just performed by calling your few-shot classifier on your query image.

```python
prediction = few_shot_classifier(query_images)
```

**Comment by shraddha291996** [comment]
> 

Hi thank you for your reply. Sorry this is new to me but here how query images should be defined. I want to make predictions on test loader in the evaluation.

**Comment by ebennequin** [comment]
`query_images` can be yielded by the dataloader just as in the notebooks.

**Comment by shraddha291996** [comment]
> `query_images` can be yielded by the dataloader just as in the notebooks.

okay got it thank you , now I get my predictions as tesnors but I am not sure how can I get the class id or class names for the predicted  ones as I want my results in class names , example I have  my test set in this format (I use Easyset) "class1" , "class2" etc. 
Can you please suggest, if there is anything I am missing?
